sdk/python/lib/pulumi/output.py

```diff
@@ -361,8 +366,9 @@ def concat(*args: List[Input[str]]) -> 'Output[str]':
         :rtype: Output[str]
         """
 
-        transformed_items = [Output.from_input(v) for v in args]
-        return Output.all(*transformed_items).apply("".join)
+        transformed_items: List[Union[Any, Awaitable[Any], Output[Any]]] = [Output.from_input(v) for v in args]
```

Comment:
isn't `Union[Any, Awaitable[Any], Output[Any]]` our way of expression `Input[Any]`.  Does that not work here?

Reply:
Good catch. Updated.